app/src/main/java/org/mifos/mobile/utils/Utils.java

```diff
@@ -63,7 +63,7 @@ public static Uri getImageUri(Context context, Bitmap bitmap) {
         File imagePath = new File(context.getCacheDir(), "images");
         File newFile = new File(imagePath, "image.png");
 
-        return FileProvider.getUriForFile(context, "org.mifos.mobilebanking.fileprovider",
+        return FileProvider.getUriForFile(context, "org.mifos.mobile.fileprovider",
```

Comment:
I think it will be better if this URL is put under strings.xml and referenced from there. What are your thoughts on this @devansh-299 ?